wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -343,6 +344,7 @@ public void stopWorkspace(String workspaceId, Map<String, String> options)
             });
   }
 
+  /** @return set of supported recipe types */
```

Comment:
According to [Google Java Code Style](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment) it makes sense to reformat this java doc as 
```
/** Returns set of supported recipe types */
```
Up to you change it or not